hack/functest.sh

```diff
@@ -4,4 +4,4 @@ set -xe
 
 source ./cluster/kubevirtci.sh
 
-KUBECONFIG=${KUBECONFIG:-$(kubevirtci::kubeconfig)} $GO test ./tests/... $E2E_TEST_ARGS -timeout=40m -ginkgo.v -test.v -race
+KUBECONFIG=${KUBECONFIG:-$(kubevirtci::kubeconfig)} $GO test ./tests $E2E_TEST_ARGS -timeout=40m -ginkgo.v -test.v -race
```

Comment:
why do we drop the `...`? I thought it was like `-r`

Reply:
ginkgo disable live stream logs if it uses recursion and since we have just one test suite here is not needed.